Jellyfin.Data/DbContexts/Jellyfin.cs

```diff
@@ -1,1140 +0,0 @@
-                     .Property(t => t.MustUpdatePassword)
-                     .IsRequired();
-         modelBuilder.Entity<global::Jellyfin.Data.Entities.User>()
-                     .Property(t => t.AudioLanguagePreference)
-                     .HasMaxLength(255)
-                     .IsRequired();
-         modelBuilder.Entity<global::Jellyfin.Data.Entities.User>()
-                     .Property(t => t.AuthenticationProviderId)
-                     .HasMaxLength(255)
-                     .IsRequired();
-         modelBuilder.Entity<global::Jellyfin.Data.Entities.User>()
-                     .Property(t => t.GroupedFolders)
-                     .HasMaxLength(65535);
-         modelBuilder.Entity<global::Jellyfin.Data.Entities.User>()
-                     .Property(t => t.InvalidLoginAttemptCount)
-                     .IsRequired();
-         modelBuilder.Entity<global::Jellyfin.Data.Entities.User>()
-                     .Property(t => t.LatestItemExcludes)
-                     .HasMaxLength(65535);
-         modelBuilder.Entity<global::Jellyfin.Data.Entities.User>()
-                     .Property(t => t.MyMediaExcludes)
-                     .HasMaxLength(65535);
-         modelBuilder.Entity<global::Jellyfin.Data.Entities.User>()
-                     .Property(t => t.OrderedViews)
-                     .HasMaxLength(65535);
-         modelBuilder.Entity<global::Jellyfin.Data.Entities.User>()
-                     .Property(t => t.SubtitleMode)
-                     .HasMaxLength(255)
-                     .IsRequired();
-         modelBuilder.Entity<global::Jellyfin.Data.Entities.User>()
-                     .Property(t => t.PlayDefaultAudioTrack)
-                     .IsRequired();
-         modelBuilder.Entity<global::Jellyfin.Data.Entities.User>()
-                     .Property(t => t.SubtitleLanguagePrefernce)
-                     .HasMaxLength(255);
-         modelBuilder.Entity<global::Jellyfin.Data.Entities.User>()
-                     .HasMany(x => x.Groups)
-                     .WithOne()
-                     .HasForeignKey("Group_Groups_Id")
-                     .IsRequired();
-         modelBuilder.Entity<global::Jellyfin.Data.Entities.User>()
-                     .HasMany(x => x.Permissions)
-                     .WithOne()
-                     .HasForeignKey("Permission_Permissions_Id")
-                     .IsRequired();
-         modelBuilder.Entity<global::Jellyfin.Data.Entities.User>()
-                     .HasMany(x => x.ProviderMappings)
-                     .WithOne()
-                     .HasForeignKey("ProviderMapping_ProviderMappings_Id")
-                     .IsRequired();
-         modelBuilder.Entity<global::Jellyfin.Data.Entities.User>()
-                     .HasMany(x => x.Preferences)
-                     .WithOne()
-                     .HasForeignKey("Preference_Preferences_Id")
-                     .IsRequired();
-
-         OnModelCreatedImpl(modelBuilder);
-      }
-   }
-}
```

Comment:
Why is this deleted?

Reply:
Replaced by `JellyfinDb`

Reply:
Right, but I think it would be better to move it instead of delete and add, so that we can get a proper diff of the changes made to it. If you do a `git mv` for it in this PR then in #2970 we'll be able to see a diff of the changes made.